tox.ini

```diff
@@ -18,12 +18,13 @@ commands=
 
 [testenv:coverage]
 deps=
-    coveralls
-    pytest-cov
-passenv= HOME GITHUB_*
+    codecov
+    coverage
 commands=
-    pytest --cov-config=gwcs/tests/coveragerc --cov=gwcs
-    coveralls --rcfile=gwcs/tests/coveragerc
+    pip freeze
+    !cov: pytest {posargs}
+    cov: pytest --cov-report xml --cov ./ {posargs}
+passenv= HOME GITHUB_* TOXENV CI CODECOV_* DISPLAY    
```

Comment:
It looks like nothing is getting uploaded to coveralls now.  Is that the intention?

Reply:
Or would you like to switch over to codecov?  Most of the other repos do use codecov, and we do have a Github action for that.

Reply:
Yes, I was thinking of switching to codecov. Can you point me to an example of that GA?

Reply:
We use Codecov's own Github action over in `jwst`:

https://github.com/spacetelescope/jwst/blob/231b4c2c2782ad1242c0fb83940b0f21cfb81ebb/.github/workflows/ci.yml#L156-L160

Ignore the flag.  And to create the coverage.xml file, we run pytest with coverage via

https://github.com/spacetelescope/jwst/blob/231b4c2c2782ad1242c0fb83940b0f21cfb81ebb/tox.ini#L42